<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainShowDialog</class>
 <widget class="QDialog" name="MaintainShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="matchIdButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>根据机电设备编号查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByTimeButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>以时间排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="idButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>170</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>根据编号查询</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>养护信息查询区</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByMatchIdButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>以机电设备编号排序</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idlineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="matchIdlineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
